src/assets/CollectiblesController.ts

```diff
@@ -430,13 +444,36 @@ export class CollectiblesController extends BaseController<
     const api = this.getCollectibleContractInformationApi(contractAddress);
     let apiCollectibleContractObject: ApiCollectibleContract;
     /* istanbul ignore if */
-    if (this.openSeaApiKey) {
-      apiCollectibleContractObject = await handleFetch(api, {
-        headers: { 'X-API-KEY': this.openSeaApiKey },
-      });
-    } else {
+
+    try {
       apiCollectibleContractObject = await handleFetch(api);
+    } catch (e) {
+      logOrRethrowError(e);
+      if (this.openSeaApiKey) {
+        apiCollectibleContractObject = await handleFetch(
+          `https://api.opensea.io/api/v1/asset_contract/${contractAddress}`,
+          {
+            headers: { 'X-API-KEY': this.openSeaApiKey },
+          },
+        );
+      } else {
+        return {
```

Comment:
Nit: this might be easier to follow of this was assigned to `apiCollectibleContractObject`, leaving only one `return` statement. Or drop that variable, and return in all 3 places.